javascript/javascript3/week2/exercise 4/DogImages/index.js

```diff
@@ -0,0 +1,11 @@
+setInterval(() => {
+    let img = document.createElement("img");
+    document.body.appendChild(img)
+    async function DogsPics() {
+     const dogName = await fetch("https://dog.ceo/api/breeds/image/random");
+     const dogNameJson = await dogName.json()
+     img.src = await dogNameJson.message
+    }
+    DogsPics()
+    }, 3000);
+    
```

Comment:
looks like the answer here

https://github.com/27ManishaPatel/hyf-homework/blob/3329c2b74ccb963901ded8c2c4b34310f3780bfb/javascript/javascript3/week2/warmup/script.js#L108

Reply:
i found it in google and study about it and did it .i think this happend by accident.i first chose an api but i couldn't  open it and then i chose another api, generally I dont use my classmates' codes becuase I dont know if they have errors or not. But i used Google a lot